src/ceph-volume/ceph_volume/devices/lvm/strategies/bluestore.py

```diff
@@ -86,14 +86,16 @@ def compute(self):
         for device in self.ssds:
             extents = lvm.sizing(device.sys_api['size'], parts=self.osds_per_device)
             for ssd in range(self.osds_per_device):
-                osd = {'data': {}, 'block.db': {}}
+                osd = {'data': {}, 'block.db': {}, 'used_by_ceph': device.used_by_ceph}
```

Comment:
I thought this was going to be:

    'filtered': {
        '/dev/sda': {'reasons': ['Used by ceph'] }
    },
    ...

I think hard coding the reason as a key is going to make it hard to be backwards compatible, and it is going to be difficult to add more reasons as a key, which will make it cumbersome for consumers of this response to translate into a meaningful message.

Reply:
I guess you are adding already the information with the format I described in args.filtered_devices. So I wonder what would the need be to set this here

Reply:
The format you described is the output that shows in ``--report``. The ``used_by_ceph`` here is the property on the ``Device`` class I use to know if that device should get filtered or not. I suppose this would always show ``False`` though because devices already used by ceph are not included there.

Reply:
But this `osd` dictionary is setting that key which I don't see being used anywhere. If these are already being filtered out then I don't see the need to have it here

Reply:
It's not needed, I'll remove it.